packages/core/admin/admin/src/components/Form.tsx

```diff
@@ -778,7 +820,7 @@ const Blocker = ({ onProceed = () => {}, onCancel = () => {} }: BlockerProps) =>
   return null;
 };
 
-export { Form, Blocker, useField, useForm, getYupValidationErrors };
+export { Form, Blocker, useField, useForm, getYupValidationErrors, FormProvider };
```

Comment:
How confident are you that you're not exporting this provider from the admin package when you do this?

Reply:
Oops I see https://github.com/strapi/strapi/blob/0c5ac9b4a0965db69613ed48a167c99c6127d9e0/packages/core/admin/admin/src/index.ts#L15-L16

🤔 wdyt the best idea is? should we do named form exports from the package index?